packages/orchestration/src/exos/zcf-for-flow.js

```diff
@@ -0,0 +1,68 @@
+import { VowShape } from '@agoric/vow';
+
+/**
+ * @import {HostOf} from '@agoric/async-flow';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ */
+
+import { M } from '@endo/patterns';
+
+export const ZcfI = M.interface(
+  'ZCF',
+  {
+    makeInvitation: M.call(M.raw(), M.string())
+      .optional(M.record(), M.pattern())
+      .returns(VowShape),
+  },
+  {
+    defaultGuards: 'passable',
+  },
+);
```

Comment:
I'm not the most familiar with interface guards, do we need to document the other methods? Should the original guards be referenced somehow?

Reply:
I copied the existing `ZcfI` and changed `.returns(M.promise())` to `.returns(VowShape)` and removed `setTestJig`.

https://github.com/Agoric/agoric-sdk/blob/914bee2aa7472039ba230267d72027b940bf8a1c/packages/zoe/src/contractFacet/typeGuards.js#L14-L25